examples/framework-astro/keystone.ts

```diff
@@ -18,7 +19,7 @@ export default config({
 
     // WARNING: this is only needed for our monorepo examples, dont do this
     //   we use using myprisma, not .myprisma, because vite
-    prismaClientPath: 'node_modules/myprisma/client',
```

Comment:
The nested nature of `/client` is not compatible with `vite`, we should probably roll-out this change to our `fixPrismaPath` utility generally